lambda-cleanup.tf

```diff
@@ -0,0 +1,36 @@
+# Trigger:
+#	Cloudwatch cron schedule
+# Resources:
+#	KMS
+#	RDS
+#	Secret manager secrets
+#	SSM parameters
+
+module "cleanup" {
+  source = "./modules/lambda"
+  enable = true
+  name   = format("%s-cleanup", module.labels.id)
+
+  aws_parameter_arns = [
+    aws_ssm_parameter.db_database.arn,
+    aws_ssm_parameter.db_host.arn,
+    aws_ssm_parameter.db_port.arn,
+    aws_ssm_parameter.db_reader_host.arn,
+    aws_ssm_parameter.db_ssl.arn,
+    aws_ssm_parameter.security_code_lifetime_mins.arn,
```

Comment:
Do we need these ?

Reply:
Yes, these two are used in the lambda code